### ISSUE TITLE
The rowNum attribute was deprecated in Matplotlib 3.2 and will be removed two minor releases later.

### ISSUE THREAD

**Issue by ikostan** [matplotlib/matplotlib#16954]
### Bug report

**Bug summary**
The rowNum attribute was deprecated warning appears when using .hist() method

**Code for reproduction**
Following code from my Google Colab

```python
# Paste your code here
# Downloading data from a URL
# source http://archive.ics.uci.edu/ml/machine-learning-databases/
!wget http://archive.ics.uci.edu/ml/machine-learning-databases/arrhythmia/arrhythmia.data
# Check if the data has a header
!head arrhythmia.data
import pandas as pd
df = pd.read_csv('arrhythmia.data', header=None, error_bad_lines=False)
# Take a first few columns and give them names
data = df[[0,1,2,3,4,5]]
data.columns = ['age', 'sex', 'height', 'weight', 'QRS duration', 'P-R interval']
import matplotlib.pyplot as plt
# Make the plot bigger so that subplots don't overlap
plt.rcParams['figure.figsize'] = [15, 15]

# use a semicolon to suppress return value
data.hist();
```

**Actual outcome**
Following warning appears:

```
# If applicable, paste the console output here
#
/usr/local/lib/python3.6/dist-packages/pandas/plotting/_matplotlib/tools.py:307: MatplotlibDeprecationWarning: 
The rowNum attribute was deprecated in Matplotlib 3.2 and will be removed two minor releases later. Use ax.get_subplotspec().rowspan.start instead.
  layout[ax.rowNum, ax.colNum] = ax.get_visible()
/usr/local/lib/python3.6/dist-packages/pandas/plotting/_matplotlib/tools.py:307: MatplotlibDeprecationWarning: 
The colNum attribute was deprecated in Matplotlib 3.2 and will be removed two minor releases later. Use ax.get_subplotspec().colspan.start instead.
  layout[ax.rowNum, ax.colNum] = ax.get_visible()
/usr/local/lib/python3.6/dist-packages/pandas/plotting/_matplotlib/tools.py:313: MatplotlibDeprecationWarning: 
The rowNum attribute was deprecated in Matplotlib 3.2 and will be removed two minor releases later. Use ax.get_subplotspec().rowspan.start instead.
  if not layout[ax.rowNum + 1, ax.colNum]:
/usr/local/lib/python3.6/dist-packages/pandas/plotting/_matplotlib/tools.py:313: MatplotlibDeprecationWarning: 
The colNum attribute was deprecated in Matplotlib 3.2 and will be removed two minor releases later. Use ax.get_subplotspec().colspan.start instead.
  if not layout[ax.rowNum + 1, ax.colNum]:
```

**Expected outcome**

Warning should not appear since I did not use "rowNum" attribute.

**Matplotlib version**

  * Operating system: Linux-4.14.137+-x86_64-with-Ubuntu-18.04-bionic
  * Matplotlib version: 3.2.1
  * Matplotlib backend (`print(matplotlib.get_backend())`): module://ipykernel.pylab.backend_inline
  * Python version: 3.6.9 (default, Nov  7 2019, 10:44:02) [GCC 8.3.0]
  * Jupyter version (if applicable):
```
jupyter core     : 4.5.0
jupyter-notebook : 5.2.2
qtconsole        : 4.5.2
ipython          : 5.5.0
ipykernel        : 4.6.1
jupyter client   : 5.3.1
jupyter lab      : not installed
nbconvert        : 5.5.0
ipywidgets       : 7.5.0
nbformat         : 4.4.0
traitlets        : 4.3.2
```
  * Other libraries: 
I used data set from here: [http://archive.ics.uci.edu/ml/machine-learning-databases/arrhythmia/arrhythmia.data](url)

**Comment by tacaswell** [comment]
Can you reproduce this issue without using pandas?

**Comment by ikostan** [comment]
Wow, I just submitted it and already have an answer... I will try. I am just a beginner so please be patient.

**Comment by ikostan** [comment]
> Can you reproduce this issue without using pandas?

Actually  I have no idea how to replace those two lines of code:
`import pandas as pd
df = pd.read_csv('arrhythmia.data', header=None, error_bad_lines=False)`

**Comment by timhoffm** [comment]
This is an issue in pandas, fixed in pandas 1.0.2:

https://github.com/pandas-dev/pandas/pull/32444

**Comment by ikostan** [comment]
> This is an issue in pandas, fixed in pandas 1.0.2:
> 
> [pandas-dev/pandas#32444](https://github.com/pandas-dev/pandas/pull/32444)

I checked pandas version and it is an old one: 0.25.0. Thank you @timhoffm
